mlflow/keras.py

```diff
@@ -759,26 +762,24 @@ def _run_and_log_function(self, original, args, kwargs, unlogged_params, callbac
 
         return history
 
-    def fit(self, *args, **kwargs):
-        original = gorilla.get_original_attribute(keras.Model, "fit")
+    def fit(original, self, *args, **kwargs):
         unlogged_params = ["self", "x", "y", "callbacks", "validation_data", "verbose"]
         return _run_and_log_function(self, original, args, kwargs, unlogged_params, 5)
 
-    def fit_generator(self, *args, **kwargs):
+    def fit_generator(original, *args, **kwargs):
         """
         NOTE: `fit_generator()` is deprecated in Keras >= 2.4.0 and simply wraps `fit()`.
         To avoid unintentional creation of nested MLflow runs caused by a patched
         `fit_generator()` method calling a patched `fit()` method, we only patch
         `fit_generator()` in Keras < 2.4.0.
         """
-        original = gorilla.get_original_attribute(keras.Model, "fit_generator")
         unlogged_params = ["self", "generator", "callbacks", "validation_data", "verbose"]
         return _run_and_log_function(self, original, args, kwargs, unlogged_params, 4)
 
-    wrap_patch(keras.Model, "fit", fit)
+    safe_patch(FLAVOR_NAME, keras.Model, "fit", fit)
```

Comment:
May want to update these patterns: `try_mlflow_log(_log_early_stop_callback_params, callback)`